terraform/aws/variables.tf

```diff
@@ -99,6 +104,11 @@ variable "web_count" {
   default = 1
 }
 
+variable "logs_enabled" {
```

Comment:
Do you handle it somewhere?

Reply:
In `instances.tf` it will only create the logs node if enabled.